servers/src/main/java/tachyon/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -92,7 +92,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
         getServletContext().getRequestDispatcher("/worker/blockInfo.jsp").forward(request,
             response);
         return;
-      } catch (NotFoundException nfe) {
+      } catch (BlockDoesNotExistException nfe) {
```

Comment:
should the exception variable name change to bdnee? 

(I personally do not see the reason for using these verbose variable names and would simply use e everywhere.)